### ISSUE TITLE
Navigate button (with restart pending) not working

### ISSUE THREAD

**Issue by bjschuit** [hacs/integration#2667]
### Web browser

Edge

### Web browser version

latest

## System Health

version | core-2022.5.0
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.9.9
os_name | Linux
os_version | 5.15.32
arch | x86_64
timezone | Europe/Amsterdam

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 4986
Installed Version | 1.24.5
Stage | running
Available Repositories | 1049
Downloaded Repositories | 9

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 8.0.dev20220404
-- | --
update_channel | stable
supervisor_version | supervisor-2022.05.0
docker_version | 20.10.12
disk_total | 232.2 GB
disk_used | 46.1 GB
healthy | true
supported | true
board | generic-x86-64
supervisor_api | ok
version_api | ok
installed_addons | AdGuard Home (4.5.1), Grafana (7.5.2), MariaDB (2.4.0), Mosquitto broker (6.1.1), Nginx Proxy Manager (0.11.0), Node-RED (11.1.2), Samba share (9.6.1), Signal Messenger (0.59.0), Studio Code Server (5.0.2), Transmission (0.7.1), UniFi Network Application (2.2.0), Vaultwarden (Bitwarden) (0.16.0), Zigbee2mqtt (1.25.1-1), InfluxDB (4.4.1), Terminal & SSH (9.4.0)

</details>

<details><summary>Dashboards</summary>

dashboards | 1
-- | --
resources | 5
views | 9
mode | storage

</details>

### Checklist

- [X] I'm running the newest version of HACS <https://github.com/hacs/integration/releases/latest>
- [X] I have filled out the issue template to the best of my ability.
- [X] I have read <https://hacs.xyz/docs/issues>
- [X] This issue is related to the frontend of HACS.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of currently [open](https://github.com/hacs/integration/issues) or issues [pending release](https://github.com/hacs/integration/issues?q=is%3Aissue+is%3Aclosed+sort%3Aupdated-desc+milestone%3Anext).

### Describe the issue

Since 2022.5 the 'Navigate' button, which appears when having a necessary restart pending, does not work anymore. Probably because of the breaking changes in the menu structure of HASS. 

### Reproduction steps

1. Update an integration of HACS.
2. Press navigate button
3. You end up on a blank page. 

### Screenshots

![image](https://user-images.githubusercontent.com/15046966/167615344-ab2ca318-352a-428b-99de-214772de23f4.png)
![image](https://user-images.githubusercontent.com/15046966/167615369-849771b4-1ac2-4db6-8ddd-db6677f652e7.png)

### Javascript logs from your browser console

```text
-
```

**Comment by bjschuit** [comment]
Apparently many duplicate issue reports. It would help if the team would follow the best practice of not closing an issue before it has actually been published. Sorry for the inconvenience.

**Comment by ludeeus** [comment]
Your statement is false.

This way is the recommended https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

You also said you had already checked before opening a new issue
![image](https://user-images.githubusercontent.com/15093472/167617170-b0aeaad1-2f36-4e00-b134-a2d88efc30b3.png)

**Comment by bjschuit** [comment]
@ludeeus I did this:
![image](https://user-images.githubusercontent.com/15046966/167622533-2184d9f8-04b2-478d-b8f5-ccb7ee679e49.png)

Because there are no OPEN issues:
![image](https://user-images.githubusercontent.com/15046966/167622695-44649e3e-4674-45ed-a184-6c1e00913c81.png)

So your statement on the recommended way is valid, but as there is no open issue anymore it is not applicable. So if you want to avoid duplicates, just keep the original issue open. I wouldn't have taken the effort (probably the same as all  other people that submitted the same issue) to mention this if I saw it was already on the radar.
